src/pages/api/post/getAvatar.ts

```diff
@@ -0,0 +1,22 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
+import { initCloudinary } from 'config/cloudinary';
+import wrongMethod from 'utils/server/wrongMethod';
+const cloudinary = require('cloudinary');
+
+initCloudinary();
+
+export default async (req: NextApiRequest, res: NextApiResponse) => {
+  if (req.method === 'GET') {
+    const images = await cloudinary.v2.search
+      .expression(`folder:nextinsta/thumbnails/${req.query.username}`)
+      .sort_by('public_id', 'desc')
+      .max_results(30);
+    console.log('Images', images);
+    const linksArray = images.resources.map(
+      (el) =>
+        `https://res.cloudinary.com/drjffngwz/image/upload/q_100,c_fill,h_293,w_293/v1653415525/${el.public_id}`
```

Comment:
This could be stored in some env file like you did with `getAvatarUrl` function.